chalk-engine-base/Cargo.toml

```diff
@@ -1,7 +1,7 @@
 [package]
-name = "chalk-macros"
+name = "chalk-engine-base"
```

Comment:
I don't particularly like the name `chalk-engine-base`. Maybe just `chalk-base`?

Reply:
Or, since `chalk-ir` is getting a feature flag. Just have this be part of `chalk-engine` with all SLG stuff under a feature flag.

Not sure how I feel about this one.

Reply:
If it's part of `chalk-engine`, 90% of the crate will be under one feature flag.

Reply:
Yeah, exactly. Which is probably not a great tradeoff to just get rid of a small crate.